content/blog/zenbleed-patch-call-for-testing.md

```diff
@@ -11,10 +11,13 @@ post:
     image: /blog-images/23.07.24.zenbleed.png
 ---
 
-Earlier today our community pointed out a new, trivially exploitable flaw in [AMD systems called Zenbleed](https://lwn.net/Articles/939099/). Due to an accident on the AMD side, the patch was released ahead of responsible disclosure, and unpatched systems are at great risk. We were able to pull in the patch, get through our normal testing, and we are now ready for wider testing for both AlmaLinux 8 and 9.
+EDIT: July 26: We will be releasing this patch on July 27th at 7am. Read more on the [forums](https://almalinux.discourse.group/t/zenbleed-patch-release-7am-eastern-us-time-7-27-23/2802).
```

Comment:
Can we post the timezone here as well?

In the future we should consider using UTC.

Reply:
```suggestion
EDIT: July 26: We will be releasing this patch on July 27th at 7am US Eastern time. Read more on the [forums](https://almalinux.discourse.group/t/zenbleed-patch-release-7am-eastern-us-time-7-27-23/2802).
```